.github/workflows/project-build.yml

```diff
@@ -13,16 +13,21 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu, macos]
-        ruby: [2.7.4, 3.0.2, 3.1.0]
-        gemfile: [activesupport_5, activesupport_6]
+        ruby: [2.7, '3.0', '3.1', '3.2']
+        gemfile: [activesupport_5, activesupport_6, activesupport_7]
+        exclude:
+          - ruby: '3.2'
+            gemfile: activesupport_5
+          - ruby: 2.7
+            gemfile: activesupport_7
     continue-on-error: ${{ endsWith(matrix.ruby, 'head') || matrix.ruby == 'debug' }}
     env: # $BUNDLE_GEMFILE must be set at the job level, so it is set for all steps
       BUNDLE_GEMFILE: gemfiles/${{ matrix.gemfile }}.gemfile
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - uses: ruby/setup-ruby@v1
         with:
           ruby-version: ${{ matrix.ruby }}
-      - run: bundle install
+          bundler-cache: true
```

Comment:
Do we need to run `bundle install`?

Reply:
@adomokos Running an explicit bundle install isn't required when using the `setup-ruby` task with `bundler-cache` set to true.